modules/plugin/src/main/scala/com/fortysevendeg/hood/plugin/SbtHoodPlugin.scala

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2019 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.fortysevendeg.hood.plugin
+
+import java.io.File
+
+import cats.data.EitherT
+import cats.implicits._
+import cats.effect.{Console, IO, Sync}
+import com.fortysevendeg.hood.benchmark.{BenchmarkComparisonResult, BenchmarkService, Warning}
+import com.fortysevendeg.hood.csv.{BenchmarkColumns, CsvService}
+import com.fortysevendeg.hood.model.{Benchmark, HoodError}
+import sbt.{AutoPlugin, Def, PluginTrigger, Task}
+import io.chrisdavenport.log4cats.Logger
+import io.chrisdavenport.log4cats.slf4j.Slf4jLogger
+import cats.effect.Console.implicits._
+
+object SbtHoodPlugin extends AutoPlugin with SbtHoodDefaultSettings with SbtHoodKeys {
+
+  override def projectSettings: Seq[Def.Setting[_]] = defaultSettings
+  override val trigger: PluginTrigger               = noTrigger
+
+  def compareBenchmarksTask: Def.Initialize[Task[Unit]] = Def.task {
+
+    implicit val logger = Slf4jLogger.getLogger[IO]
```

Comment:
We could initialize this logger as part of the plugin initialization. We may just move it at object level.